Vince

vforgione opened the pull request.

Commit:
Clean Slate

Commit:
The Basics

Commit:
Added Filters

Commit:
Added Tests

Commit:
AoTClient and Various Abstraction Cleans Ups

- Added a new AoT Client to the clients
- Added new ABC bases
- Cleaned up client requests
- Cleaned up tests

brlodi commented:
Quick note, you blew away the license file in the first commit and I assume you want to put that back.

vforgione commented:
Closes #5 
Closes #6 
Closes #7 
Closes #8 
Closes #9

Commit:
Readded the License

sanilio reviewed (changes requested):
Looks great, I suggest we change `class Response` to `PlenarioResponse` to as it shadows `requests.Response`

Commit:
Changed Response to PlenarioResponse

Having a class named `Response` could be confusing to users, so it's a good idea to explicitly name it.

Commit:
Fixed Renaming in Tests
